### PR TITLE
docs: Document `version` csv format

### PR DESCRIPTION
**Pull request by vvoland**
- follow up to: https://github.com/docker/setup-docker-action/pull/106

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,14 +112,70 @@ jobs:
...
+| `version`       | String | `latest`              | Docker version to use. See [inputs.version](#inputs.version).                                                                |
+| `channel`       | String | `stable`              | Docker CE [channel](https://download.docker.com/linux/static/) (`stable` or `test`). Only applicable to `type=archive`       |
+| `daemon-config` | String |                       | [Docker daemon JSON configuration](https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-configuration-file)  |
+| `context`       | String | `setup-docker-action` | Docker context name.                                                                                                         |
+| `set-host`      | Bool   | `false`               | Set `DOCKER_HOST` environment variable to docker socket path.                                                                |
+| `rootless`      | Bool   | `false`               | Start daemon in rootless mode                                                                                                |
+
+### inputs.version
```
Comment (crazy-max) [inline README.md:124]: add this to TOC after https://github.com/docker/setup-docker-action/blob/2a248d822ca74a4cc67d26b4e33fe7b9f9cf2422/README.md?plain=1#L33

File: README.md [hunk 2]
```diff
@@ -112,14 +112,70 @@ jobs:
...
+for each branch and PR, so you can use the `tag` input to install a specific version or branch (e.g. `master`).
+
+See https://hub.docker.com/r/moby/moby-bin/tags and https://hub.docker.com/r/dockereng/cli-bin/tags for available tags.
+
+| Type       | Key       |  Default   | Description                                                                          |
+|------------|-----------|------------|--------------------------------------------------------------------------------------|
+|            | `type`    | `archive`  | The source type of the Docker binaries. Possible values are `archive` and `image`.   |
+| `archive`  | `channel` | `stable`   | The download.docker.com channel (`stable` or `test`).                                |
+| `archive`  | `version` | `latest`   | The Docker version to use.                                                           |
+| `image`    | `tag`     | `latest`   | The image tag to use.                                                                |
```
Comment (crazy-max) [inline README.md:147]: nit: got a bit confused at first when looking at this table. Maybe we could split by type like we do in metadata-action repo: https://github.com/docker/metadata-action?tab=readme-ov-file#tags-input

**Review by crazy-max — COMMENTED** [review]
PTAL @dvdksn

**Review by crazy-max — APPROVED** [review]

**Review by dvdksn — COMMENTED** [review]
oh 🤦🏻 I was too slow to submit my comments. @crazy-max I can do a follow-up PR if that's OK with you?

**Comment by crazy-max** [comment]
> oh 🤦🏻 I was too slow to submit my comments. @crazy-max I can do a follow-up PR if that's OK with you?

Ah sorry I merged too fast, please feel free to open a PR :pray:

**Pull request merged**